src/pychart/_interpreter/ast_nodes/statement.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Optional, Any
+
+from src.pychart._interpreter.ast_nodes.expression import Expr
+from src.pychart._interpreter.token_type.token import Token
+
+
+class Stmt:
+    def __call__(self, visitor: "StmtVisitor") -> Any:
+        raise RuntimeError("Expected Actual Statement")
+
+
+class StmtVisitor:
+    @staticmethod
+    def throw():
+        raise Exception("Unimplmented visitor method")
+
+    # pylint: disable=unused-argument
+    def expression(self, stmt: "Expression") -> Any:
+        StmtVisitor.throw()
+
+    def print(self, stmt: "Print") -> Any:
+        StmtVisitor.throw()
+
+    def let(self, stmt: "Let") -> Any:
+        StmtVisitor.throw()
+
+    def block(self, stmt: "Block") -> Any:
+        StmtVisitor.throw()
+
+    # pylint: enable=unused-argument
+
+
+class Expression(Stmt):
+    expr: Expr
+
+    def __init__(self, expr: Expr):
+        self.expr = expr
+
+    def __call__(self, visitor: StmtVisitor) -> Any:
+        return visitor.expression(self)
+
+
+class Print(Stmt):
+    expr: Expr
+
+    def __init__(self, expr: Expr):
+        self.expr = expr
+
+    def __call__(self, visitor: StmtVisitor) -> Any:
+        return visitor.print(self)
+
+
+class Let(Stmt):
+    name: Token
+    initializer: Optional[Expr]
```

Comment:
Does this imply that:
```
let a;
```
is legal, it think it would be more clear if a value is required, especially if we are still trying to make a language used for learning.